joomla/joomla-cms
# [5.0] schemaorg fix user parameter

### Summary of Changes

- inject `UserFactory` into schemaorg system plugin
- get user plugin parameter (`userId` is used instead of `user`)
https://github.com/joomla/joomla-cms/blob/d0a2509484366bf4439b8266c1debfe3e88d1e7f/plugins/system/schemaorg/schemaorg.xml#L37-L39

### Testing Instructions

set base type to `Person` and select an user account
![image](https://github.com/joomla/joomla-cms/assets/66922325/7a71d785-1781-40dd-9be0-9ddf8c01e47e)

check Person type

### Actual result BEFORE applying this Pull Request

`name` is not present (or you entered the name manually and to not match to the user)
![image](https://github.com/joomla/joomla-cms/assets/66922325/e466946f-bf46-42f3-8119-71d0c9f03f24)

### Expected result AFTER applying this Pull Request

`name` is preset (`name` of the selected user)
![image](https://github.com/joomla/joomla-cms/assets/66922325/af95201f-21c3-4979-96cd-1f6dcf8b1844)

### Link to documentations
Please select:
